pyproject.toml

```diff
@@ -45,6 +45,12 @@ tox = "^3.24.5"
 tox-poetry = "^0.4.1"
 pyright = "^0.0.13"
 typing-extensions = "^4.0.1"
+flake8 = "^4.0.1"
+flake8-bugbear = "^22.1.11"
+flake8-tidy-imports = "^4.6.0"
+flake8-import-order = "^0.18.1"
+pep8-naming = "^0.12.1"
+flake8-annotations = "^2.7.0"
```

Comment:
Out of all the things that flake8 sorts, does it ignore these?

Reply:
You mean you want flake8-import-order to sort the order of dependencies in pyproject.toml? Flake8 is a linter for python files, I don't believe there's any support for any other fotmat, maybe there's a plugin that changes that, but it's probably pretty hacky at best, flake8 was never meant to lint more than python files

Reply:
It's okay.